src/openforms/js/lang/nl.json

```
@@ -49,6 +49,11 @@
     "description": "Registration backend options label",
     "originalDefault": "Registration backend options"
   },
+  "2uIYst": {
+    "defaultMessage": "The process definition for which to start a process instance.",
```

Make my life easier by not having to search for untranslated texts...

```suggestion
    "defaultMessage": "De procesdefinitie waarvoor een proces instantie gestart moet worden.",
```